app/src/main/java/com/sopt/androidstudy/presentation/compose/ComponentActivity.kt

```diff
@@ -0,0 +1,203 @@
+            ToDoType.HANG_OUT,
+            "제주도 여행",
+            LocalDateTime.of(2022, 12, 18, 15, 0)
+        ),
+
+        ToDoTask(
+            ToDoType.STUDY,
+            "밀린 강의 청산",
+            LocalDateTime.of(2022, 12, 27, 20, 0)
+        ),
+    )
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        setContent {
+            AndroidSturdyTheme {
+                Surface(
+                    modifier = Modifier.fillMaxSize(),
+                    color = MaterialTheme.colors.background
+                ) {
+                    val selectDate = remember {
+                        mutableStateOf<Int?>(null)
+                    }
+                    val currentMonth = remember {
+                        mutableStateOf(LocalDateTime.now().month.value)
+                    }
+
+                    Column() {
+                        Row(
+                            horizontalArrangement = Arrangement.SpaceEvenly, modifier = Modifier
+                                .padding(0.dp, 12.dp, 0.dp, 20.dp)
+                                .fillMaxWidth()
+                        ) {
+                            Text(
+                                text = "이전",
+                                fontSize = 20.sp,
+                                color = Color.Black,
+                                modifier = Modifier
+                                    .border(
+                                        width = 1.dp,
+                                        color = Color.Gray,
+                                        shape = RectangleShape
+                                    )
+                                    .padding(10.dp)
+                                    .clickable {
+                                        selectDate.value = null
+                                        currentMonth.value = currentMonth.value.dec()
+                                    }
+                            )
+                            Text(
+                                text = currentMonth.value.toString(),
+                                fontSize = 20.sp,
+                                color = Color.Black
+                            )
+                            Text(
+                                text = "다음",
+                                fontSize = 20.sp,
+                                color = Color.Black,
+                                modifier = Modifier
+                                    .border(
+                                        width = 1.dp,
+                                        color = Color.Gray,
+                                        shape = RectangleShape
+                                    )
+                                    .padding(10.dp)
+                                    .clickable {
+                                        selectDate.value = null
+                                        currentMonth.value = currentMonth.value.inc()
```

Comment:
```suggestion
currentMonth.value = currentMonth.value.inc().takeIf { it <= 12 } ?: currentMonth.value
```
현재 12월에서 다음을 눌렀을 때 앱이 crash가 됩니다 
따라서, 위와 같이 나타내줄 수 있을 것 같습니다

Reply:
일부러 모든 케이스가 안 되고 이번년도까지만 되게 만들어놨습니다! 모든 케이스 안할거면 달을 안 움직이게 하는 방법도 있겠네요! 감사합니당